wordpress-mobile/WordPress-iOS
# WordPressAuthenticator asume https

When the site address for log in has no scheme, the app will asume https instead of http

This PR fixes a wrong error message when the server blocks http requests and it doesn't redirect to https.

To test:

- Go to `Log in by site address`
- Enter a magic address that has this problem (without specifying scheme)
- **Before**: It was giving an error: `Not a WP site`
- **Now**: It should let you go to the user/password screen
